src/App.tsx

```diff
@@ -15,31 +15,65 @@ import { theme } from "./theme";
 
 library.add(fas, far);
 
-function App() {
-  return (
-    <MuiThemeProvider theme={theme}>
-      <div className={styles.App}>
-        <Header />
-        <div className={styles.app_content_wrapper}>
-          <Router>
-            <Route exact path="/" component={Pages.MainPage} />
-            <Route
-              exact
-              path="/projects/create"
-              component={Pages.CreateProjectPage}
-            />
-            <Route
-              path="/projects/detail/:uuid"
-              component={Pages.ProjectDetailPage}
-            />
-          </Router>
-        </div>
-        <div className={styles.footerPosition}>
-          <Footer />
+// ログインしているユーザーの情報
+type AppStatus = {
+  loginUserFollowingIdList: number[];
+};
+
+class App extends React.Component<{}, AppStatus> {
```

Comment:
空のPropsを定義してからComponentの型引数に渡すべき？
```suggestion
type AppProps = {}

class App extends React.Component<{}, AppStatus> {
```